docs/content/preview/yugabyte-voyager/reference/data-migration/export-data.md

```diff
@@ -99,8 +133,72 @@ The valid *arguments* for export data status are described in the following tabl
 | -e, --export-dir <path> | Path to the export directory. This directory is a workspace used to store exported schema DDL files, export data files, migration state, and a log file.|
 | -h, --help | Command line help. |
 
-### Example
+#### Example
 
 ```sh
 yb-voyager export data status --export-dir /dir/export-dir
 ```
+
+### get data-migration-report
+
+Provides a consolidated report of data migration per table among all the databases involved in the live migration. The report includes the number of rows exported, the number of rows imported, change events exported and imported (INSERTS, UPDATES, and DELETES), and the final row count on the database.
+
```

Comment:
Add in the starting, `For live migration only, it provides. a consolidated ...`

Reply:
Adding a general note at the beginning of each live migration command on export and import data cli pages .